backend/Virkarekisteri/src/Functions/PositionName/GetPositionNames.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+using Virkarekisteri.Repositories;
+
+namespace Virkarekisteri.Functions.PositionName
+{
+    public class GetPositionNames
```

Comment:
Could use a simpler constructor here, similar to CreatePosition function. @0x464e mentioned this before in another PR

Reply:
done